chrome_manifest_v3/views/popupView.js

```
@@ -214,19 +216,19 @@ Gmail2Trello.PopupView.prototype.confirmPopup = function () {
  * "Add card" button to the edge of the window and then center that under the "Add card" button:
  */
 Gmail2Trello.PopupView.prototype.centerPopup = function (useWidth) {
-  var g2tLeft = this.$g2tButton.position().left;
-  var g2tRight = g2tLeft + this.$g2tButton.width();
-  var g2tCenter = g2tLeft + this.$g2tButton.outerWidth() / 2;
+  let g2tLeft = this.$g2tButton.position().left;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'g2tLeft' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)